port/win/port_win.h

```diff
@@ -16,22 +16,22 @@
 #define WIN32_LEAN_AND_MEAN
 #endif
 
-#include <windows.h>
-#include <string>
-#include <thread>
-#include <string.h>
-#include <mutex>
-#include <limits>
-#include <condition_variable>
-#include <malloc.h>
 #include <intrin.h>
+#include <malloc.h>
```

Comment:
The compile error on windows is that ['_aligned_malloc' was not declared](https://app.circleci.com/pipelines/github/facebook/rocksdb/24377/workflows/7161371a-4e84-46f3-8038-df8f52ace6ef/jobs/569179), but I have already included the header file here.

Reply:
Bad things can happen when `#include <windows.h>` is not first. See mmap.h comment.

Reply:
@pdillinger I tried to keep the original order, but then the clang-format complains that the files are not in sorted order.

Reply:
You should be able to work around it with blank lines or interrupting line comments like in mmap.h